libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -57,4 +60,22 @@ export class BookSearchComponent {
       this.store.dispatch(clearSearch());
     }
   }
+
+  private showSnackBar(book: Book) {
+    const snackBar = this.snackBar.open( "Added" + ' ' + book.title + ' ' + 'to Reading list!', 'Undo', {
+      duration: 1000
```

Comment:
I think that 1 second is a very small timeframe.. Maybe increase it a bit?

Reply:
Increased time frame.